kolibri_instant_schools_plugin/assets/src/views/UserIndex.vue

```diff
@@ -1,22 +1,22 @@
 <template>
 
-  <CoreBase
+  <AppBarPage
```

Comment:
I am not sure this will work? You probably need to conditionalize whether you're using the AppBarPage or the just a div depending on the conditions below instead?

Noting that in the current user auth plugin in Kolibri we don't use either the AppBarPage or ImmersivePage: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/user_auth/assets/src/views/UserAuthIndex.vue